crates/uv/src/commands/pip/compile.rs

```diff
@@ -547,6 +547,12 @@ fn cmd(
                 return Some(None);
             }
 
+            // Always skip the `--upgrade-package` flag, and mark the next item as skipped
+            if arg == "--upgrade-package" || arg == "-P" {
```

Comment:
Looks like this does not work with `--upgrade-package=` or `-P=`. I should have looked more carefully at `--find-links` above.

Will create a fix.

Reply:
Thanks!